tests/component/encapsulation.cpp

```diff
@@ -96,23 +96,75 @@ TEST(Encapsulation, reparentComponent)
 
     model->addComponent(parent);
 
+    // model (parent (child1, child2, child3 ) )
+
     libcellml::Printer printer;
     std::string a_parent = printer.printModel(model);
     EXPECT_EQ(e_parent_1, a_parent);
 
-    // what do we expect this to achieve? The addition of child3 to child2
+    // what do we expect this to achieve? The addition of child3 to child2 << KRM swaps parents of child3 but does not update component() array
+
+    std::cout << "parent component has children: \n";
```

Comment:
For all `std::cout` it should be `EXPECT_*`.

Reply:
Missed this one - will comment it all out for now.  It shows the bug in how we handle parents.